### PR TITLE
Add Network csproj to CosmosDB sln

### PR DESCRIPTION
**Pull request by soeom**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by dingmeng-xue** [comment]
@VeryEarly , please check sln change.

**Comment by soeom** [comment]
@VeryEarly  Can you help merge this PR?

File: src/CosmosDB/CosmosDB.sln [hunk 1]
```diff
@@ -6,6 +6,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CosmosDB", "CosmosDB\Cosmos
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{95C16AED-FD57-42A0-86C3-2CF4300A4817}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Network", "..\Network\Network\Network.csproj", "{98CFD96B-A6BC-4F15-AE2C-603FC2B58981}"
+EndProject
```
Comment (VeryEarly) [inline src/CosmosDB/CosmosDB.sln:10]: please follow line 32 - lin 37, and add {98CFD96B-A6BC-4F15-AE2C-603FC2B58981} after line 37

Comment (soeom) [reply src/CosmosDB/CosmosDB.sln:10]: Fixed. Thanks.

**Comment by soeom** [comment]
@VeryEarly Please help merge the PR

**Comment by VeryEarly** [comment]
network was added by #11592 
This PR can be closed.

**Pull request closed without merging**